src/otx/algo/detection/atss.py

```diff
@@ -95,12 +67,14 @@ def __init__(
     def _create_model(self) -> nn.Module:
         from mmengine.runner import load_checkpoint
 
-        config = deepcopy(self.config)
-        self.classification_layers = self.get_classification_layers()
-        model = TorchATSS(**convert_conf_to_mmconfig_dict(config))
+        detector = self._build_model(num_classes=self.label_info.num_classes)
+        self.classification_layers = self.get_classification_layers(prefix="model.")
```

Comment:
This might be out of scope of this PR, but can I ask you what `self.classification_layers` is for?

Reply:
It is for getting name of final classification layer name. The classification layer name is used during smart weight loading.

Reply:
Is there any possibility of drop this concept?

Reply:
I hardly think so, could you share your concern?

Reply:
Let's discuss in another session